2/src/styles/_action-box.scss

```diff
@@ -0,0 +1,67 @@
+.action-box {
+  display: grid;
+  grid-template-columns: repeat(2, minmax(auto, 40%));
+  gap: 20%;
+  max-width: 1030px;
+  background: linear-gradient(
+    to right top,
+    $gradientColor2 10%,
+    darken($gradientColor1, 10%) 40%
+  );
+  color: white;
+  margin: -60px auto 80px auto;
+  z-index: 10;
+
+  h4 {
+    border-bottom: 1px solid white;
+    padding: 30px 10px 10px 0;
+    font: {
+      family: Acme, sans-serif;
+      size: 12px;
+      weight: 400;
+    }
+    letter-spacing: 2px;
+  }
+
+  & > div:first-of-type {
```

Comment:
the same as above 
also try selector .class + .class { padding-left } it is faster

Reply:
done

Reply:
done